feature-guide.md

```diff
@@ -56,19 +99,26 @@ In order to implement a new unstable feature, you need to do the following steps
 1. Open a [tracking issue](#tracking-issues) - if you have an RFC, you can use the tracking issue for the RFC.
 2. Pick a name for the feature gate (for RFCs, use the name in the RFC).
 3. Add a feature gate declaration to `libsyntax/feature_gate.rs`
-    In the active `declare_features` block:
+    In the active `declare_features` block:  
     ```
     // description of feature
     (active, $feature_name, "$current_nightly_version", Some($tracking_issue_number))
     ```
-    For example
+    
+    For example  
```

Comment:
nit: if you're adding a hard break to the previous block, might as well add one here

Reply:
what's a hard break?

Reply:
The extra two spaces at the end of the line. It makes markdown add a `<br>` instead of joining the lines together.

Reply:
But there is a hard break after "For example"

Reply:
Oh my bad, i didn't see it. `>_>`